### ISSUE TITLE
Conflicting peer dependencies with `@logux/eslint-config` and `eslint-config-standard`

### ISSUE THREAD

**Issue by vrizo** [logux/eslint-config#23]
Hello @ai,

I’m trying to install `@logux/eslint-config` in my new project. It seems like there are some peer dependency conflicts involving `eslint-plugin-n` and `eslint-plugin-promise` from `eslint-config-standard`.

Here are the specific conflicts:

- `@logux/eslint-config@53.5.1` installs **eslint-plugin-n@^17.13.2** (currently **17.15.1**), but  
  `eslint-config-standard@17.1.0` requires **eslint-plugin-n@^15.0.0 \|\| ^16.0.0**.  

- `@logux/eslint-config@53.5.1` installs **eslint-plugin-promise@^7.1.0** (currently **7.2.1**), but  
  `eslint-config-standard@17.1.0` requires **eslint-plugin-promise@^6.0.0**.

Just in case: I’m using `npm` (not pnpm), but I’m not sure if that matters.

I’ve tried cleaning the npm cache and fixing permissions after recent issue in npm, but the installation still fails due to these version conflicts. I would be happy to create a PR, but I’m unsure how to fix this, so asking you to check when you have time :) 

Not urgent since it works well even with these warnings.

<details>
  <summary>Full NPM log with the error</summary>

  ```bash
    % npm install --save-dev @logux/eslint-config eslint
    npm warn ERESOLVE overriding peer dependency
    npm warn While resolving: eslint-config-standard@17.1.0
    npm warn Found: eslint-plugin-n@17.15.1
    npm warn node_modules/eslint-plugin-n
    npm warn   eslint-plugin-n@"^17.13.2" from @logux/eslint-config@53.5.1
    npm warn   node_modules/@logux/eslint-config
    npm warn     dev @logux/eslint-config@"*" from the root project
    npm warn
    npm warn Could not resolve dependency:
    npm warn peer eslint-plugin-n@"^15.0.0 || ^16.0.0 " from eslint-config-standard@17.1.0
    npm warn node_modules/eslint-config-standard
    npm warn   eslint-config-standard@"^17.1.0" from @logux/eslint-config@53.5.1
    npm warn   node_modules/@logux/eslint-config
    npm warn
    npm warn Conflicting peer dependency: eslint-plugin-n@16.6.2
    npm warn node_modules/eslint-plugin-n
    npm warn   peer eslint-plugin-n@"^15.0.0 || ^16.0.0 " from eslint-config-standard@17.1.0
    npm warn   node_modules/eslint-config-standard
    npm warn     eslint-config-standard@"^17.1.0" from @logux/eslint-config@53.5.1
    npm warn     node_modules/@logux/eslint-config
    npm warn ERESOLVE overriding peer dependency
    npm warn While resolving: eslint-config-standard@17.1.0
    npm warn Found: eslint-plugin-promise@7.2.1
    npm warn node_modules/eslint-plugin-promise
    npm warn   eslint-plugin-promise@"^7.1.0" from @logux/eslint-config@53.5.1
    npm warn   node_modules/@logux/eslint-config
    npm warn     dev @logux/eslint-config@"*" from the root project
    npm warn
    npm warn Could not resolve dependency:
    npm warn peer eslint-plugin-promise@"^6.0.0" from eslint-config-standard@17.1.0
    npm warn node_modules/eslint-config-standard
    npm warn   eslint-config-standard@"^17.1.0" from @logux/eslint-config@53.5.1
    npm warn   node_modules/@logux/eslint-config
    npm warn
    npm warn Conflicting peer dependency: eslint-plugin-promise@6.6.0
    npm warn node_modules/eslint-plugin-promise
    npm warn   peer eslint-plugin-promise@"^6.0.0" from eslint-config-standard@17.1.0
    npm warn   node_modules/eslint-config-standard
    npm warn     eslint-config-standard@"^17.1.0" from @logux/eslint-config@53.5.1
    npm warn     node_modules/@logux/eslint-config
  ```
</details>

**Comment by ai** [comment]
Unfortunately, nothing we can do about it until Standard will release new version.

Maybe it is a time to migrate somewhere from Standard (I am open for suggestions).

**Comment by vrizo** [comment]
Thanks for your quick reply! Since it doesn't block the usage of this configuration, I believe it's better to wait to save time and resources for other priorities.

**Comment by ai** [comment]
Fixed in 55.0 by moving to [Neostandard](https://github.com/neostandard/neostandard)
